tasks/tor.yml

```diff
@@ -20,6 +20,16 @@
   register: descriptor_cookie
   when: ssh_onion and torrc.changed
 
+- name: Grab sandstorm onion address
+  shell: cat /var/lib/tor/sandstorm/hostname | awk '{print $1}'
+  register: sandstorm_onion_address
+  when: sandstorm_onion and torrc.changed
+
+- name: Grab sandstorm client descriptor cookie
```

Comment:
Oh, good point.  That should be removed.  If someone actually wanted this on http, they would know to add it themselves, and it's nonfunctional as it is.